worlds/sm64ex/Items.py

```diff
@@ -7,8 +7,9 @@ class SM64Item(Item):
     "Power Star": 3626000,
     "Basement Key": 3626178,
     "Second Floor Key": 3626179,
-    "Wing Cap": 3626180,
-    "Metal Cap": 3626181,
-    "Vanish Cap": 3626182,
-    "1Up Mushroom": 3626183
+    "Progressive Key": 3626180,
+    "Wing Cap": 3626181,
```

Comment:
I'm concerned about the effects of changing item IDs of existing items. Wouldn't this make new game builds incompatible with multiworlds generated on the previous version?

Reply:
Yes. But that's the case anyway